tests/Browser/UserSearchTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use Tests\Browser\Pages\UserSearchPage;
+use Tests\Browser\Pages\HomePage;
+
+class UserSearchTest extends DuskTestCase
+{
+    /**
+     * Test visiting the User Search Page.
+     * Searching for a user by name (instead of
+     * valid email address) returns "No user found!" message.
+     */
+    public function testIncorrectUserSearchPage()
+    {
+        // $this->browse(function (Browser $browser) {
+        //     $browser->login()
+        //         ->visit(new UserSearchPage)
+        //         ->assertSeeIn('@title', 'Users');
+                // ->type('@search', 'taylor')
+                // ->press('Submit')
+                // ->assertSeeIn('@messages', 'No user found!');
+        // });
+
+        $this->browse(function (Browser $browser) {
+            $browser->visit(new HomePage)
+                    ->click('@login-button')
+                    ->dump();
+                    // ->assertPathIs('https://northstar-qa.dosomething.org/register');
```

Comment:
sorry @DFurnes - another q! When I dump here, it gives me this error: 
```
."<html xmlns="http://www.w3.org/1999/xhtml"><head></head><body><pre style="word-wrap: break-word; white-space: pre-wrap;">{"error":"unsupported_grant_type","message":"The authorization grant type is not supported by the authorization server.","hint":"Check the `grant_type` parameter"}</pre></body></html>"
```
I don't think I need to be sending a token or anything in the header at this point since I should just be redirected to https://northstar-qa.dosomething.org/register but maybe I'm wrong?

Reply:
We need to make sure we have the correct `NORTHSTAR_AUTH_ID` and `NORTHSTAR_AUTH_SECRET` set here (in our `.env.dusk.*` files) so Northstar knows we're allowed to log users in.

We'll also need to figure out what redirect URL this is setting when making the `/authorize` request, and add that to Northstar QA as a valid `redirect_uri`... you can get that from `$browser->driver->getCurrentURL()`.

Reply:
ah ok I will play around with that - thank you!